src/cpu/pred/2bit_local.cc

```diff
@@ -67,15 +79,15 @@ LocalBP::LocalBP(const LocalBPParams &params)
 }
 
 void
-LocalBP::btbUpdate(ThreadID tid, Addr branch_addr, void * &bp_history)
+LocalBP::updateHistories(ThreadID tid, Addr pc, bool uncond,
+                         bool taken, Addr target, void * &bpHistory)
 {
-// Place holder for a function that is called to update predictor history when
-// a BTB entry is invalid or not found.
+// Place holder for a function that is called to update predictor history
 }
 
 
 bool
-LocalBP::lookup(ThreadID tid, Addr branch_addr, void * &bp_history)
+LocalBP::lookup(ThreadID tid, Addr branch_addr, void * &bpHistory)
```

Comment:
Can we revert the changes from `bpHistory` back to `bp_history` in this and all the files where this change is made?

I think `bp_history` better conforms to the gem5 style guide: https://www.gem5.org/documentation/general_docs/development/coding_style/